### DISCUSSION TITLE
RGB GeoTIFF Bands compostion

### DISCUSSION THREAD

**Discussion by alexisahedo** [ESA-PhiLab/OpenSarToolkit#86] [Q&A] [answered]
Hello dear everyone,

First, I’ll like to express my gratitude to all who have made this excellent code possible, it has been very helpful to me.

I’ll like to ask a question I have had in mind for while in regard with the **RGB** **GeoTIFF** output file, as the examples explain:
“Sentinel-1 scenes usually consist of two polarization bands. In order to create a 3 channel RGB composite a ratio between the Co- (VV or HH) and the Cross-polarized (VH or HV) band is added. The create_rgb method takes the ard_dimap file and converts it to a 3-channel GeoTiff.”

I’ll like to confirm if the **Band 1** is Co (VV or HH), **Band 2** is Cross-polarization (VH or HV) and **Band 3** is the ratio of mentioned bands (Band 1/Band 2, which doesn’t seem the case). Could you please help me clear this confusion? Displaying in QGIS gives little information of the metadata in this regard. 

Thanks a lot for your help.

Alexis A

**Comment by BuddyVolly** [comment, marked as answer]
Dear Alexis, 

yes, that should be the case. However it is the ratio in power scale, so calculated before the co and cross pol bands have been converted to the decibel scale. In the decibel scale, the actual operation would be to subtract one from the other. 

Hope that clarifies it,
Andreas
